modules/smart-agent_kubernetes-workloads-count/conf/readme.yaml

```
@@ -0,0 +1,22 @@
+documentations:
+  - name: Kubernetes Deployment
+    url: 'https://kubernetes.io/docs/concepts/workloads/controllers/deployment/'
+source_doc: >-
+  We are using metrics from the
+  [SignalFX Support for Kubernetes](https://docs.signalfx.com/en/latest/integrations/agent/agent-k8s-install-helm.html)
+notes: |
+  ### Kubernetes Deployment Metrics
+  The default threshold values are:
+    * critical > 20
+    * major > 15 and < 20
+  If those values are not relevants, please feel free to set your own threshold.
```

What I looking for is an explanation of the purpose of the detector instead of default thresholds which is a redundant information available in variables values (from the registry).

```suggestion
This detector allows to set a custom limit on the total count of workloads configured on a Kubernetes cluster. This represents the sum of deployments, replicasets, statefulsets and daemonsets desired.

It can allow you set a capacity limit and be warned when it has been reached.
```